src/renderer/src/elements/main-dashboard.ts

```diff
@@ -500,13 +508,25 @@ export class MainDashboard extends LitElement {
       }
     });
 
-    // add eventlistener for pocket
+    // add eventlistener for search and pocket
     window.addEventListener('keydown', (zEvent) => {
       if (zEvent.altKey && zEvent.key === 's') {
         // case sensitive
-        switch (this.showClipboard) {
+        switch (this.showSearch) {
           case false:
-            this.showClipboard = true;
+            this.showSearch = true;
+            this._searchDialog.show('open');
+            this._searchDialog.focus();
+            break;
+          case true:
+            this._searchDialog.hide();
+            break;
+        }
+      } else if (zEvent.altKey && zEvent.key === 'p') {
```

Comment:
I can't press `Alt + P` with a single hand on my keyboard...I wonder whether we could choose a keyboard combination that's easy to press with a single hand.

Reply:
Alt-e or Alt-t?